azure/packages/azure-client/src/AzureClient.ts

```diff
@@ -231,12 +232,12 @@ export class AzureClient {
 
```

Comment:
the changes below here are a good example of the new usages, we use factory functions + interfaces, rather than leaking classes. all the code changes are similar.